tests/js/realm-tests.js

```diff
@@ -18,11 +18,8 @@
 
 'use strict';
 
-/* global navigator, WorkerNavigator */
-
-const require_method = require;
-
 // Prevent React Native packager from seeing modules required with this
+const require_method = require;
 function nodeRequire(module) {
     return require_method(module);
 }
```

Comment:
```suggestion
const nodeRequire = require;
```

Reply:
this wont work

Reply:
Why not? I believe I've used that elsewhere and it looks equivalent to me. Is it picked up by the RN bundler?